presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -432,18 +391,77 @@ node.
+Property Value                                     Description
+================================================== ============================================================
+``legacy`` (default value)                         Few authorization checks are enforced, thus allowing most
+                                                   operations. The config properties ``hive.allow-drop-table``,
+                                                   ``hive.allow-rename-table``, ``hive.allow-add-column``,
+                                                   ``hive.allow-drop-column`` and
+                                                   ``hive.allow-rename-column`` are used.
+
+``read-only``                                      Operations that read data or metadata, such as ``SELECT``,
+                                                   are permitted, but none of the operations that write data or
+                                                   metadata, such as ``CREATE``, ``INSERT`` or ``DELETE``, are
+                                                   allowed.
+
+``file``                                           Authorization checks are enforced using a config file specified
+                                                   by the Hive configuration property ``security.config-file``.
+                                                   See :ref:`hive-file-based-authorization` for details.
+
+``sql-standard``                                   Users are permitted to perform the operations as long as
+                                                   they have the required privileges as per the SQL standard.
+                                                   In this mode, Presto enforces the authorization checks for
+                                                   queries based on the privileges defined in Hive metastore.
+                                                   To alter these privileges, use the :doc:`/sql/grant` and
+                                                   :doc:`/sql/revoke` commands.
+                                                   See :ref:`hive-sql-standard-based-authorization` for details.
+
+``ranger``                                         Users are permitted to perform the operations as per the
+                                                   authorization policies configured in Ranger Hive service.
+                                                   See :ref:`hive-ranger-based-authorization` for details.
+================================================== ============================================================
+
+.. _hive-sql-standard-based-authorization:
+
+SQL Standard Based Authorization
+--------------------------------
+
+When ``sql-standard`` security is enabled, Presto enforces the same SQL
+standard based authorization as Hive does.
+
+Since Presto's ``ROLE`` syntax support matches the SQL standard, and
```

Comment:
```suggestion
Because Presto's ``ROLE`` syntax support matches the SQL standard and
```
Minor suggestion here: Since has meaning of both time and cause, Because means only cause.